### PR TITLE
Add Taxpayer Identification Number validation

### PR DESCRIPTION
**Pull request by franciscocardoso**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import { default as isValidEin, mask as maskEin }  from 'is-valid-ein';
+import { default as isValidItin, mask as maskItin } from 'is-valid-itin';
+import { default as isValidSsn, maskSSN as maskSsn } from 'is-valid-ssn';
+
+/**
+ * Export isValid function.
+ */
+
+export function isValid(tin) {
```
Comment (promag) [inline src/index.js]: rename to `validate`?

**Comment by nunofgs** [comment]
Can we get this merged "as-is" guys?

I can submit a PR later to improve the module a bit but I'd like to use this as a dependency.

/cc @franciscocardoso, @ruimarinho

**Comment by promag** [comment]
Use https://github.com/seegno/eslint-config-seegno?

**Comment by nunofgs** [comment]
@promag: as I said, I can do that in a later PR.

As it stands, this module does not have a single tag to which I can point to, when declaring it as a dependency.

I'd love to just merge this, tag it as 0.0.1 and then I'll submit a PR to clean it up to our usual standards.

**Comment by franciscocardoso** [comment]
It would be cool to to that with all the dependencies aswell.

**Comment by franciscocardoso** [comment]
:+1:

**Comment by nunofgs** [comment]
Landed in 0511c86.

**Pull request closed without merging**